blobfuse/blobfuse.cpp

```diff
@@ -215,6 +217,10 @@ int read_config(const std::string configFile)
         fprintf(stderr, "No config file found at %s.\n", configFile.c_str());
         return -1;
     }
+    else
+    {
+       fprintf(stderr, "reading config");
```

Comment:
This will print out as an error/to the console, so do we want this to be printing out whenever a configuration file is being read? If we find a configuration file we shouldn't be printing out anything

Reply:
done